examples/configs/onnx_infer.yml

```diff
@@ -0,0 +1,82 @@
+# defaults:
+#   - _self_
+#   - base_configparams
+
+task:
+  name: ONNXTask
+  params:
+    path_to_onnx: '/workdir/vpatrushev/small2/resnet18-v2-7.onnx'
```

Comment:
This is an example, don't put hard-coded paths. Also, I suggest to make this example a follow-up of the CIFAR10 training example mentioned in README. And put the command to README